src/Symfony/Component/Messenger/Attribute/Senders.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Attribute;
+
+/**
+ * @author Maxim Dovydenok <dovydenok.maxim@gmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class Senders
```

Comment:
AFIAK we get the sender from a transport, so perhaps `#[Transport(name)]`?

also i tend to prefer repeatable singulars, for more granularity and potential better composition